src/indicator/algorithm/merge.js

```diff
@@ -1,16 +1,15 @@
-import calculator from './calculator/slidingWindow';
-import d3 from 'd3';
-import {noop} from '../../util/fn';
+import { zip } from 'd3-array';
```

Comment:
I think we're sticking with d3 in d3fc for now, any chance you could change this over to `d3.zip`?